app/classifier/tasks/highlighter/index.jsx

```diff
@@ -36,6 +37,23 @@ export default class Highlighter extends React.Component {
     selection.collapseToEnd();
   }
 
+  getOffset(selection) {
+    const { anchorNode } = selection;
+    const { parentNode } = anchorNode;
+    let start = 0;
+    let counting = true;
+    parentNode.childNodes.forEach((node) => {
+      // ignore comments (nodeType 8) when calculating offset distance
+      if (counting && node.nodeType !== 8 && node !== anchorNode) {
+        start += node.textContent.length;
+      }
+      if (node === anchorNode) {
+        counting = false;
```

Comment:
Sadly, `NodeList.forEach` isn't supported in Edge or IE.
https://developer.mozilla.org/en-US/docs/Web/API/NodeList